Automattic/pocket-casts-android
# Added a Halloween icon

This change adds a new app Halloween icon. 

Another change I made was to resize the images for the Red Velvet and Radioactivity in the settings appearance page. The code has a max width for the images of 124dp in the recycler view adapter. This change reduced the apk size from 23.9MB to 22.8MB.

## Test steps

1. Make you use have Plus
2. Tap the Profile tab
3. Tap the Setting cog
4. Tap Apperance
5. Swipe the "App Icon" list to see the Halloween icon at the end of the list
6. Tap the Halloween icon
7. Quit the app
8. Swipe up into the list of apps
9.  ✅ Verify the Halloween app icon is visible

## Screenshots

<img  width="300" src="https://user-images.githubusercontent.com/308331/196573652-e49517b4-a040-4f1b-a0cd-82a658cca578.png" />
<img  width="300" src="https://user-images.githubusercontent.com/308331/196573657-e655f0fa-33f7-4af1-81cf-0e88eba790c8.png" />
<img  width="300" src="https://user-images.githubusercontent.com/308331/196590257-f4e3de64-015f-4cb1-81c9-b6109f2d1aec.png" />
